scripts/testing/rebuild_boot_iso

```diff
@@ -21,26 +21,31 @@ Usage:
 
 Options:
  -h, --help                         Print this help
+ -a, --add-rpms                     Add additional RPMs (for example anaconda-webui package)
+                                    these packages needs to be already requested in ISO
```

Comment:
I think there is a typo - should be just `"need"`. Also I suggest `"already requested to be in the ISO"`